recipes/apriltag/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "apriltag" %}
+{% set version = "3.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AprilRobotics/apriltag/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: eef3ab9283c4d8a92a06b40e468ae98726f5d485b888309babefb8092a2e544e
+
+build:
+  number: 0
+
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage('apriltag', max_pin='x.x.x') }}
```
Please add a run export so downstream users of the C API will depend on an ABI compatible version of libapriltag.so. Adjust the max_pin expression according to ABI compatability.

https://docs.conda.io/projects/conda-build/en/stable/resources/variants.html#extra-jinja2-functions